pdns/fstrm_logger.cc

```diff
@@ -9,7 +9,8 @@
 
 #ifdef HAVE_FSTRM
 
-FrameStreamLogger::FrameStreamLogger(const int family, const std::string& address, bool connect): d_family(family), d_address(address)
+FrameStreamLogger::FrameStreamLogger(const int family, const std::string& address, bool connect,
+    const std::unordered_map<string,unsigned>& options): d_family(family), d_address(address)
```

Comment:
I'm not very fond of passing a map of options to a constructor, I'd rather see setters if we have too many of them. Please feel free to disagree, though :)

Reply:
The problem is that the `FrameStreamLogger` create the framestream connection on construct. Setters would be too late. Options:
* rework the FrameStreamLogger with a three step setup (construct; set setters; connect)
* use a settings object: `frameStreamSettings.setOption(x)....; FrameStreamLogger(... frameStreamSettings);`

Reply:
Ah, right, I missed that indeed! Let's keep it that way for now, then.